package/src/components/StripePaymentCheckoutAction/v1/StripePaymentCheckoutAction.js

```diff
@@ -99,14 +103,20 @@ class StripePaymentCheckoutAction extends Component {
      * this function.
      */
     onSubmit: PropTypes.func,
+    /**
+     * The text for the "Your Information is private and secure." caption text.
+     */
+    secureCaptionText: PropTypes.string,
     /**
      * Checkout process step number
      */
     stepNumber: PropTypes.number.isRequired
   };
 
   static defaultProps = {
-    onReadyForSaveChange() { }
+    billingAddressTitleText: "Billing Address"
+    onReadyForSaveChange() { },
```

Comment:
Seems that this comma is causing problems

Reply:
woops will fix

Reply:
done